### PR TITLE
ssh control file check for bonding test

### PR DESCRIPTION
**Pull request by vaishnavibhat**
Bug fix for issue
https://github.com/avocado-framework-tests/avocado-misc-tests/issues/2032

Signed-off-by: Vaishnavi Bhat <vaishnavi@linux.vnet.ibm.com>

### CODE REVIEW

**Comment by vaishnavibhat** [comment]
avocado run bonding.py -m bonding.py.data/bonding_single.yaml
JOB ID     : 2318d20dff96315c3f2a2a210ae2e9a345e0b2fd
JOB LOG    : /root/avocado-fvt-wrapper/results/job-2021-04-19T05.10-2318d20/job.log
 (1/3) bonding.py:Bonding.test_setup;run-1fe2: PASS (23.54 s)
 (2/3) bonding.py:Bonding.test_run;run-1fe2: PASS (237.54 s)
 (3/3) bonding.py:Bonding.test_cleanup;run-1fe2: PASS (27.98 s)
RESULTS    : PASS 3 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB HTML   : /root/avocado-fvt-wrapper/results/job-2021-04-19T05.10-2318d20/results.html
JOB TIME   : 290.04 s

[bonding.log](https://github.com/avocado-framework-tests/avocado-misc-tests/files/6334870/bonding.log)

File: io/net/bonding.py [hunk 1]
```diff
@@ -112,6 +112,11 @@ def setUp(self):
         self.peer_wait_time = self.params.get("peer_wait_time", default=5)
         self.sleep_time = int(self.params.get("sleep_time", default=5))
         self.mtu = self.params.get("mtu", default=1500)
+        for root, dirct, files in os.walk("/root/.ssh"):
+            for file in files:
+                if file.startswith("avocado-master-root"):
```
Comment (abdhaleegit) [inline io/net/bonding.py:117]: when user changes, the file name changes too.. example `avocado-master-hscroot` ... make it generic something like `avocado-master`

Comment (abdhaleegit) [reply io/net/bonding.py:117]: same for padmin

**Review by abdhaleegit — CHANGES_REQUESTED** [review]
Please make the file name generic

File: io/net/bonding.py (hunk 1 above)
Comment (abdhaleegit) [reply io/net/bonding.py:117]: `file` is a builtin in python,  but it works fine on python3.

**Review by abdhaleegit — APPROVED** [review]
LGTM

**Pull request merged**